cpu/stm32/bootloader/Makefile

```
@@ -0,0 +1,3 @@
+MODULE = bootloader_stm32
```

```suggestion
MODULE = stm32_bootloader
```

I'd prefer if the module is named `<SUBSYSTEM>_<NAME>` if the subsystem is part of the name.

---

Using `bootloader_xx` follows the convention used for other bootloader already supported by the build system, e.g. `bootloader_arduino`, `bootloader_nrfutil`. So I would keep `bootloader_stm32` here.

---

Good argument! Better keep it consistent :-)